governance/polls/Ratification Poll - MIP40c3-SP23 - August 09, 2021.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Ratification Poll for Growth Core Unit MKR Budget - GRO-001 (MIP40c3-SP23) - August 09, 2021
+summary: MIP40c3-SP23 adds the MKR Incentive Plan budget for Core Unit GRO-001: GROWTH.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp23-growth-core-unit-mkr-budget-gro-001/9245
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-08-09T16:00:00
+end_date: 2021-08-23T16:00:00
+---
+# Ratification Poll for Growth Core Unit MKR Budget - GRO-001 (MIP40c3-SP23) - August 09, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, August 09 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP23 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This proposal contains:
+
+Total MKR Expenditure Cap
+The evaluation process to define the total team MKR per year.
+Estimated MKR Expenditure
+Payment Implementation
+Our proposal took elements from the others presented before, and we added an achievement metric that modifies the total team MKR per year. This is a 3-year vesting plan with a 1-year cliff vest.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/commit/06d8190480cfe45f57f44312c9193960bd1a4d69)
```

Comment:
Closer here, the actual link should be: https://github.com/makerdao/mips/blob/06d8190480cfe45f57f44312c9193960bd1a4d69/MIP40/MIP40c3-Subproposals/MIP40c3-SP23.md